packages/formation-react/src/components/Pagination/Pagination.jsx

```diff
@@ -17,16 +18,30 @@ class Pagination extends Component {
 
   static defaultProps = {
     ariaLabelSuffix: '',
+    disableGATracking: false,
```

Comment:
It's kinda confusing having `disable` and defaulted to `false` because it's like two inverse operations. I think a `enableGATracking` defaulted to `true` might be a little simpler

Reply:
For sure, that makes more sense! I'll make the update 🙂